src/main/java/blusunrize/immersiveengineering/api/client/ICacheKeyProvider.java

```diff
@@ -0,0 +1,25 @@
+package blusunrize.immersiveengineering.api.client;
```

Comment:
Hm, not sure what's causing that. At one point I intentionally removed a comment (since the file was part of BMS), but that doesn't explain why it's missing everywhere else.